CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
```

Comment:
[Question] Is this supposed to be 'may', or 'should'? Another way to phrase it: is there a circumstance in which not applying a label would be considered fine?

Reply:
hmm. good question. *may* in this context means one of many. I don't see a circumstance where /all/ of the available labels could be applied concurrently. (or, rather, such a circumstance would violate scoping issues...)

Further, by using *may* that grants permission for reviewers to append tags without explicitly requiring them to do so.

It makes more sense for me to use `may` rather than `should`. `Should` denotes the submittor **must** apply labels, and also compels reviewers to do the same; which i feel is too forceful in this context.

Reply:
The latter paragraph answers my question, thank you.